packages/core/src/sass/app/modules/account-signup.scss

```
@@ -35,47 +35,34 @@
             margin-left: unset;
         }
     }
-    &__residence-selection,
-    &__password-selection {
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        flex-flow: nowrap column;
-        background-color: var(--general-main-2);
-        border-radius: 1rem;
-        position: relative;
-        left: 50%;
-        transform: translate(-50%);
-
+    &__main {
         @include mobile {
-            width: calc(90vw - 10px);
-            padding: 6rem 4rem;
-            background-color: var(--general-main-2);
-        }
-    }
-    &__same-citizenship {
-        display: flex;
-        justify-content: space-between;
-        &-buttons {
-            display: flex;
-            gap: 1rem;
+            padding: 9rem 1.6rem;
         }
-    }
-    .dc-password-input {
-        &__visibility-icon {
-            background-color: var(--general-main-2);
-        }
-    }
-    &__residence-selection {
         .dc-select-native {
-            margin-bottom: 2.5rem;
+            margin-bottom: 0.2rem;
         }
-        @include mobile {
-            height: 100%;
-            & .dc-select-native {
-                margin-bottom: 4.8rem;
+        .account-signup__residence-selection {
+            height: 8.6rem;
+            margin-bottom: 1rem;
+            .account-signup__subtext {
+                margin-left: 1.6rem;
+                margin-top: 1rem;
+            }
+            .dc-field--error {
+                margin-left: 1.6rem;
+                margin-top: 1rem;
```

```suggestion
            .account-signup__subtext {
                margin-left: 1.6rem;
                margin-top: 1rem;
            }
            .dc-field--error {
                margin-left: 1.6rem;
                margin-top: 1rem;
```
this can be a single className shared with these two.